examples/grpc-kotlin/src/test/kotlin/example/armeria/grpc/kotlin/HelloServiceTest.kt

```diff
@@ -25,156 +19,104 @@ class HelloServiceTest {
 
     @Test
     fun reply() {
-        val helloService = Clients.newClient(uri(), HelloServiceBlockingStub::class.java)
-        assertThat(helloService.hello(HelloRequest.newBuilder().setName("Armeria").build()).message)
-                .isEqualTo("Hello, Armeria!")
+        runBlocking {
+            val helloService = Clients.newClient(uri(), HelloServiceCoroutineStub::class.java)
+            assertThat(helloService.hello(HelloRequest.newBuilder().setName("Armeria").build()).message)
+                    .isEqualTo("Hello, Armeria!")
+        }
     }
 
-    // Should never reach here.
     @Test
-    fun replyWithDelay() {
-        val helloService = Clients.newClient(uri(), HelloServiceFutureStub::class.java)
-        val future = helloService.lazyHello(HelloRequest.newBuilder().setName("Armeria").build())
-        val completed = AtomicBoolean()
-        Futures.addCallback(future, object : FutureCallback<HelloReply> {
-            override fun onSuccess(result: HelloReply?) {
-                assertThat(result?.message).isEqualTo("Hello, Armeria!")
-                completed.set(true)
-            }
-
-            override fun onFailure(t: Throwable) { // Should never reach here.
-                throw Error(t)
-            }
-        }, MoreExecutors.directExecutor())
+    fun replyWithGJson() {
+        runBlocking {
+            val uri = "gjson-web+http://127.0.0.1:" + server.activeLocalPort() + '/'
+            val helloService = Clients.newClient(uri, HelloServiceCoroutineStub::class.java)
+            assertThat(helloService.hello(HelloRequest.newBuilder().setName("Armeria").build()).message)
+                    .isEqualTo("Hello, Armeria!")
+        }
+    }
```

Comment:
There is an intermittent test failure in `gjson-web` with coroutine. 
The response was hanging and waiting infinitely. I cannot reproduce it with gRPC-Java BlockingStub or Stub with `StreamObserver`. 🤔